deployment/helm-repo/index.yaml

```diff
@@ -1,9 +1,95 @@
 apiVersion: v1
 entries:
   united-manufacturing-hub:
+  - apiVersion: v2
+    appVersion: 0.9.10-prerelease1
```

Comment:
unintended, right?

Reply:
This is going to be overwritten once this pr is merged into main after 0.9.10 is merged into main. Worst case a merge conflict raises and is easily solvable